components/audit_table/audit_table.jsx

```diff
@@ -218,16 +216,13 @@ function AuditTable(props) {
     const {formatMessage} = props.intl;
     for (var i = 0; i < props.audits.length; i++) {
         const audit = props.audits[i];
```

Comment:
Looks like `props.getByName` and `props.getUser` are being called on render function.

I'm imagining that each `audit` should be handled by a new component (maybe `AuditItem`) where user and channel object will be handled by its container.